.github/workflows/query-list.yml

```diff
@@ -0,0 +1,39 @@
+name: Build code scanning query list
+
+on:
+  push:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Clone self (github/codeql)
+      uses: actions/checkout@v2
+      with:
+        path: codeql 
+    - name: Clone github/codeql-go
+      run: git clone https://github.com/github/codeql-go
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Download CodeQL CLI
+      uses: dsaltares/fetch-gh-release-asset@master
```

Comment:
My preference would be to avoid a dependency on an external Action, and do this by directly calling https://developer.github.com/v3/repos/releases/#get-a-release-asset.

Bash step equivalent:
```
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
run: |
  DOWNLOAD_URL=$(gh api -X GET repos/github/codeql-cli-binaries/releases/latest | jq -r '.assets | map(select(.name == "codeql-linux64.zip"))[0].url')
  gh api -X GET -H accept:application/octet-stream "$DOWNLOAD_URL" > codeql-linux64.zip
```

If you wish to stick with the external Action, please pin the Action version to a specific full-length SHA, and do not use a branch reference like `master`.

Reply:
I've now specified the full SHA. I think using the external Action is fine for this 'MVP'; if we decide to offer this functionality in a different way (e.g. integrated into CodeQL CLI) in the future, then we can reconsider.